src/Core/Parser/TemplateParser.php

```diff
@@ -588,6 +588,11 @@ public function unquoteString($quotedValue)
     protected function textAndShorthandSyntaxHandler(ParsingState $state, $text, $context)
     {
         $sections = preg_split(Patterns::$SPLIT_PATTERN_SHORTHANDSYNTAX, $text, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
+        if ($sections === false) {
```

Comment:
Would it make sense to also check `preg_last_error()` in this case?

Reply:
Probably not because we cannot work with the result unless it's an array.

Reply:
It would definitely make sense - if we had any logging capabilities. I considered outputting the failure reason along with the captured text but it's not really possible to safely add arbitrary HTML comment or output since this might happen in any inline syntax including an argument.